scripts/run_testnet.sh

```diff
@@ -52,39 +53,26 @@ stop_network() {
     done
 
     echo "Session keys added. Stopping network..."
-    check_lock_file "bootnode"
+    check_lock_file "node-9944"
     check_lock_file "node-9955"
     check_lock_file "node-9966"
 }
 
-run_bootnode() {
-    local rpc_port="$1"
-
-    "$node" \
-        --chain testnet \
-        --force-authoring \
-        --rpc-cors=all \
-        --validator \
-        --rpc-port $1 \
-        --base-path "${base_path}/bootnode/" \
-        --node-key 0000000000000000000000000000000000000000000000000000000000000001 &
-
-    network_pids+=($!)
-}
-
 run_node() {
     local rpc_port="$1"
     local p2p_port="$2"
 
     "$node" \
-        --chain testnet \
+        --chain "$chainspec_path" \
         --force-authoring \
         --rpc-cors=all \
         --validator \
-        --rpc-port $rpc_port \
+        --state-pruning archive \
+        --allow-private-ipv4 \
+        --rpc-port "$rpc_port" \
         --base-path "${base_path}/node-${rpc_port}/" \
-        --port $p2p_port \
-        --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp &
+        --unsafe-force-node-key-generation \
```

Comment:
Automatically generate node keys: this is not recommended for testnet/mainnet to security risks.
Can we avoid using --unsafe-force-node-key-generation by securely generating keys manually?

For example using a script:
`
/node-data key generate-node-key --file /noda-data/node-key
`